Index: cppreference-doc/build_link_map.py
===================================================================
--- cppreference-doc.orig/build_link_map.py
+++ cppreference-doc/build_link_map.py
@@ -38,7 +38,7 @@ def build_link_map(directory):
     link_map = LinkMap()
 
     for fn in html_files:
-        f = open(fn, "r")
+        f = open(fn, "r", encoding='utf-8')
         text = f.read()
         f.close()
 
